krabs/krabs/etw.hpp

```diff
@@ -262,9 +262,18 @@ namespace krabs { namespace details {
     EVENT_TRACE_LOGFILE trace_manager<T>::fill_logfile()
     {
         EVENT_TRACE_LOGFILE file = {};
-        file.LoggerName          = const_cast<wchar_t*>(trace_.name_.c_str());
-        file.ProcessTraceMode    = PROCESS_TRACE_MODE_EVENT_RECORD |
-                                   PROCESS_TRACE_MODE_REAL_TIME;
+
+        if (!trace_.logFilename_.empty())
```

Comment:
nit: let's maintain the original alignment on the `=` characters:

```
        if (!trace_.logFilename_.empty())
        {
            file.LogFileName         = const_cast<wchar_t*>(trace_.logFilename_.c_str());
            file.ProcessTraceMode    = PROCESS_TRACE_MODE_EVENT_RECORD;
        }
        else
        {
            file.LoggerName          = const_cast<wchar_t*>(trace_.name_.c_str());
            file.ProcessTraceMode    = PROCESS_TRACE_MODE_EVENT_RECORD |
                                       PROCESS_TRACE_MODE_REAL_TIME;
        }
```

Reply:
Will do.